services/app-api/handlers/otel_handler.ts

```diff
@@ -0,0 +1,23 @@
+import opentelemetry from "@opentelemetry/api";
+import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node";
+import { Resource } from "@opentelemetry/resources";
+import { OTLPTraceExporter} from "@opentelemetry/exporter-trace-otlp-grpc"
+import { SimpleSpanProcessor, ConsoleSpanExporter } from "@opentelemetry/sdk-trace-base";
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```

Comment:
what is setting this env var for?

Reply:
@mojotalantikite I think we put this in while debugging the New Relic piece.  Given your proxy work, will we still need this for logging to NR to work?